federator/src/contracts/IBridgeNft.ts

```diff
@@ -0,0 +1,27 @@
+export class IBridgeNft {
+  nftBridgeContract: any;
+
+  constructor(nftBridgeContract: any) {
+    this.nftBridgeContract = nftBridgeContract;
+  }
+
+  getFederation(): any {
```

Comment:
Try to add types when you can, specially the Promise if they are async.
All addresses are returned as strings, numbers are returned as BigNumbers from BN.js
Hashes are returned as strings
```suggestion
  getFederation(): Promise<string> {
```

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/257/commits/960a77254356ad684efe45071b291b518e0266f8